executor/builder.go

```diff
@@ -1583,11 +1581,6 @@ func (b *executorBuilder) getSnapshotTS() (uint64, error) {
 // and some stale/historical read contexts. For example, it will return txn.StartTS in RR and return
 // the current timestamp in RC isolation
 func (b *executorBuilder) getReadTS() (uint64, error) {
```

Comment:
Will this `getReadTS` be refactored in the future so that the `ts` is returned from some statement provider or context provider? There's also a `GetStmtReadTS` in the transaction manager.

Reply:
Yes, after refactor completed, this will be removed